app/views/early_career_payments/claims/ineligible.html.erb

```diff
@@ -6,9 +6,11 @@
 
     <%= render "ineligibility_reason_#{current_claim.eligibility.ineligibility_reason}" %>
 
-    <p class="govuk-body">
-      If you want to know more about who can apply, check our
-      <%= link_to "eligibility page", EarlyCareerPayments.eligibility_page_url, class: "govuk-link" %>.
-    </p>
+    <% if current_claim.eligibility.ineligibility_reason != :ineligible_nqt_in_academic_year_after_itt %>
```

Comment:
@martikat I think you might have mentioned this to me. Did we check with UCD team if this is required? I am looking at the V10 prototype and cannot see this on the error page. Could you double check please (if you have not already done so)

Reply:
This is to hide this text as it is not used on the page. The prototype does not show this text for this ineligibility reason page